src/views/LayoutWithRouting.tsx

```diff
@@ -55,26 +59,38 @@ const LayoutWithRouting: React.FC = () => {
   }, [])
   return (
     <>
-      <GlobalStyle additionalStyles={globalStyles} />
+      <GlobalStyle additionalStyles={[globalStyles, routingTransitions]} />
       <TopNavbar default />
       <SideNavbar items={SIDENAVBAR_ITEMS} />
       <MainContainer>
-        <Router primary={false}>
-          <Route default Component={HomeView} />
-          <Route path={routes.video()} Component={VideoView} />
-          <Route path={routes.search()} Component={SearchView} />
-          <Route path={routes.videos()} Component={VideosView} />
-          <Route path={routes.channels()} Component={ChannelsView} />
-          <Route path={routes.channel()} Component={ChannelView} />
-        </Router>
+        <Location>
+          {({ location }) => (
+            <TransitionGroup>
```

Comment:
There are 2 problems here:
1) The transition sometimes looks bad when two views overlap. The transitions in Figma actually fade out the current view and only then start fading in the new one. I think it will look better without the overlap. (if not feasible, we will need to adjust entry/exit timing to alleviate
2) The old view sometimes error out. Go to some channel page, open console and switch to some different view. An exception will be logged